homeassistant/components/frontier_silicon/config_flow.py

```diff
@@ -0,0 +1,239 @@
+"""Config flow for Frontier Silicon Media Player integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.components import ssdp
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PORT
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import (
+    CONF_PIN,
+    CONF_WEBFSAPI_URL,
+    DEFAULT_PIN,
+    DEFAULT_PORT,
+    DOMAIN,
+    SSDP_ATTR_SPEAKER_NAME,
+    SSDP_ST,
+)
+from .device_validation import (
+    CannotConnect,
+    InvalidAuth,
+    validate_device_config,
+    validate_device_url,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): str,
+        vol.Required(CONF_PORT, default=DEFAULT_PORT): int,
+    }
+)
+
+STEP_DEVICE_CONFIG_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(
+            CONF_PIN,
+            default=DEFAULT_PIN,
+            description="Can be found via 'MENU button > Main Menu > System setting > Network > NetRemote PIN setup'",
```

Comment:
Not sure if this is allowed, since this cannot be translated like this. The only way I know how to add this, is to add it as description key in the strings.json, but than it cannot be linked directly to a field I think.

Reply:
Moved it to the description of the step instead